web-docs/walkthrough/index.adoc

```diff
@@ -0,0 +1,369 @@
+Save it in the same folder as your process.
+
+=== Download and Install the Seed Command Line Interface (CLI)
+
+You will find the latest version of Seed here: {seed-releases-url}
+
+* Scroll down and click on the green-outlined button on the left that says “Latest Release”
+* Scroll down to “Assets” and download the Seed version for your operating system. Note: “darwin” is the correct version for macOS.
+
+For more detailed instructions, refer to the Seed {seed-install-url}[installation instructions].
+
+You should now be able to run seed from any directory.
+
+=== Create a Dockerfile
+
+A Dockerfile is a set of instructions for what needs to be done to run your process. 
+Use a text editor to create it in the same directory as your process and name it simply “Dockerfile” (without any suffix).
+
+For a complete list of Dockerfile directives, see the {docker-reference-url}[Dockerfile reference].
+
+As an example, here is a Dockerfile to accompany the `input-output.py` file from Step 1.
+
+.Dockerfile
+[source]
+----
+FROM alpine:3.9
+
+RUN apk -U add python
+
+COPY ./input-output.py /app
+
+WORKDIR /app
+
+ENTRYPOINT ["python", "input-output.py"]
+----
+
+The Dockerfile defined in this example takes the following steps:
+
+* Start `FROM` the pre-existing `alpine:3.9` Linux image. This is an official image, validated by Docker.
+* `RUN` the command `apk -U add python` inside your image filesystem, which will install Python.
+* `COPY` the file input-output.py to your work directory.
+* Use `WORKDIR` to create the `/app` directory and specify that all subsequent actions should be taken from within `/app` _in your image filesystem_ (never the host’s filesystem).
+* Use `ENTRYPOINT` to describe how to run the container. In this case, it will run `python input-output.py`. The arguments will be specified at runtime.
+
+=== Create a Seed Manifest
+
+* Go back to your terminal window. 
+* Navigate to the directory containing your process.
+* Create a generic seed manifest file named “seed.manifest.json” by running: `seed init`
+* Edit the seed manifest file to customize it to your algorithm. 
+
+[TIP]
+====
+See the Seed {seed-spec-url}[specification] for detailed information on each property.
+====
+
+Here is an example of the seed manifest file for our input-output.py file:
+
+.seed.manifest.json
+[source,json]
+----
+{
+  "seedVersion": "1.0.0",
+  "job": {
+    "name": "io-demo",
+    "jobVersion": "1.0.0",
+    "packageVersion": "1.0.0",
+    "title": "Input/Output Demonstration",
+    "description": "Reads in a text file, changes the filename, and saves an identical copy to the destination directory.",
+    "tags": [
+      "input",
+      "output"
+    ],
+    "maintainer": {
+      "name": "Nikkala Thomson",
+      "organization": "AIS",
+      "email": "nikkala.thomson@appliedis.com"
+    },
+    "timeout": 3600,
+    "interface": {
+      "command": "${INPUT_FILE} ${OUTPUT_DIR}",
```

Comment:
Related to the entry point comment from above we should put the executable in here.

Reply:
See above.